tests/unit/tests_suite_test.go

```diff
@@ -16,6 +20,43 @@ const (
 	helmReleaseName             = "k8ssandra-test"
 )
 
+// Common labels for k8ssandra chart
+func GetK8ssandraRequiredLabels() map[string]string {
+
+	k8ssandraChart, err := chartutil.Load(chartsPath)
+	Expect(err).ToNot(BeNil())
+	Expect(k8ssandraChart).ToNot(BeNil())
+
+	meta := k8ssandraChart.Metadata
+	requiredLabels := map[string]string{
+		"helm.sh/chart":                "k8ssandra-" + meta.Version,
+		"app.kubernetes.io/name":       "k8ssandra",
+		"app.kubernetes.io/instance":   helmReleaseName,
+		"app.kubernetes.io/version":    meta.AppVersion,
+		"app.kubernetes.io/managed-by": "Helm",
+		// TODO - once we merge with changes in #197, add this one as expected
+		// "app.kubernetes.io/part-of: ": helmReleaseName+"/"+defaultTestNamespace,
+	}
+	return requiredLabels
+}
+
+func GetK8ssandraTemplates(k8ssandraChartPath string) []string {
```

Comment:
Will this get the templates that are rendered by child charts? The following components are or will be moved into separate charts:

* cass-operator
* reaper-operator
* medusa-operator
* Stargate
* Reaper

Reply:
Correct.  This is setup for an existing layout of ./templates for a given chart.  Reusable, but would need a higher-level test driver in the commons.   This was written some time ago now, just after the chart consolidation in k8ssandra.